src/noisepy/seis/plotting_modules.py

```diff
@@ -50,26 +50,28 @@ def plot_waveform(
 
     USAGE:
     -----------------------
-    plot_waveform(raw_data_store, '2023-10-21T00:00:00+0000 - 2023-10-21T12:00:00+0000', 'CI','BLC',0.01,0.5)
+    plot_waveform(raw_data_store, '2015-03-22T10:00:00+0000 - 2015-03-22T10:10:00+0000', 'CI','BLC',0.01,0.5)
     """
     # get all available timespans
     timespans = raw_data_store.get_timespans()
     if ts not in timespans:
-        raise ValueError("no data for timestamp: %s in %s" % (ts, timespans))
+        raise ValueError("no data for timespan: %s in %s" % (ts, timespans))
```

Comment:
It occurred to me that depending on the store the list of timespans could be really long, so printing all out can be too much. Instead we can print it as a range:
```suggestion
        raise ValueError("no data for timespan: %s in [%s-%s]" % (ts, timespans[0].start_datetime, timespans[-1].end_datetime))
```

Reply:
Done